demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -935,6 +937,29 @@ def is_valid_parameters_display_name(self) -> bool:
 
         return True
 
+    def is_valid_integration_file_path(self) -> bool:
+        absolute_file_path = self.file_path
+        integrations_folder = os.path.dirname(absolute_file_path).split('/')[-1]
```

Comment:
I think there should be a way to use `os` only to get the dir name without the split - the reason being that this does not support windows in this way.

Reply:
done